python/GafferUI/WidgetAlgo.py

```diff
@@ -48,3 +54,14 @@ def joinEdges( listContainer ) :
 	for i in range( 0, l ) :
 		visibleWidgets[i]._qtWidget().setProperty( lowProperty, i > 0 )
 		visibleWidgets[i]._qtWidget().setProperty( highProperty, i < l - 1 )
+
+def grab( widget, imagePath ) :
+
+	GafferUI.EventLoop.waitForIdle()
```

Comment:
Can we get this out of here, and just rely on the screen grab app to do it? It doesn't feel like an inherent part of taking a screen grab...